### PR TITLE
[#Feature: 162727510] Login Component

### PR DESCRIPTION
**Pull request by ebenezerdon**
#### What does this PR do?
- This PR adds a login component

#### Description of Task to be completed?
- Users should be able to Login to Learnground

#### How should this be manually tested?
- navigate to `/login`
- you should be able to login with correct credentials

#### What are the relevant pivotal tracker stories?
- [#162727510](https://www.pivotaltracker.com/story/show/162727510)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,16 +34,14 @@
...
     "express": "^4.16.4",
     "file-loader": "^3.0.1",
     "google-fonts-webpack-plugin": "^0.4.4",
     "html-webpack-plugin": "^3.2.0",
     "materialize-css": "^1.0.0-rc.2",
     "node-sass": "^4.11.0",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0",
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0",
```
Comment (rajeman) [inline package.json:44]: @ebenezerdon Is there any reason why you changed the versions of the react and react-dom packages?

Comment (andela-mabdussalam) [reply package.json:44]: The caret(^) symbol before the packages updates packages to the most recent major version when we run `npm install`

**Comment by andela-mabdussalam** [comment]
Since we have a navbar now, clicking on `Sign in` on the navbar should redirect to the signin page.

Additionally, add more test to cover for `operations.js` in the login directory. You can take a cue from the signup tests.

File: package.json (hunk 1 above)
Comment (XROLE) [reply package.json:44]: Ok. Thanks for clarifying. Though I was thinking that the version number would increase to say `16..9.0` as against `16.7.0` is there a reason for that please?

Comment (andela-mabdussalam) [reply package.json:44]: I suspect `16.8.0` was the latest major version the last time he ran `npm install`.

**Comment by ebenezerdon** [comment]
Thanks, I've made the changes.

File: src/app/enzymeSetup.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { configure } from 'enzyme';
```
Comment (andela-mabdussalam) [inline src/app/enzymeSetup.js]: This file is duplicated. Which of them do we want to retain?

Comment (ebenezerdon) [reply src/app/enzymeSetup.js]: I've moved the testSetup file to the `src` folder.

**Review by andela-mabdussalam — APPROVED** [review]

**Pull request merged**